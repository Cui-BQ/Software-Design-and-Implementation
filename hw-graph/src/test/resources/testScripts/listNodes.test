# List node for graphs that have 0, 1 and 2 nodes.

# Create graphs
CreateGraph graph0
CreateGraph graph1
CreateGraph graph2

# Add 1 node to graph1
AddNode graph1 n1

# Add 2 nodes to graph2
AddNode graph2 n1
AddNode graph2 n2

# graph0 lists nodes
ListNodes graph0

# graph1 lists nodes
ListNodes graph1

# graph2 lists nodes
ListNodes graph2