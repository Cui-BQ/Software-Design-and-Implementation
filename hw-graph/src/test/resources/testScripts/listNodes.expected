# List node for graphs that have 0, 1 and 2 nodes.

# Create graphs
created graph graph0
created graph graph1
created graph graph2

# Add 1 node to graph1
added node n1 to graph1

# Add 2 nodes to graph2
added node n1 to graph2
added node n2 to graph2

# graph0 lists nodes
graph0 contains:

# graph1 lists nodes
graph1 contains: n1

# graph2 lists nodes
graph2 contains: n1 n2